### PR TITLE
Aahel/new test fix

### PR DESCRIPTION
**Pull request by aahel**
Changes proposed in this PR:
-
-

How I've tested this PR:

How I expect reviewers to test this PR:

Checklist:
- [ ] Tests added
- [ ] [CHANGELOG entry added](https://github.com/hashicorp/consul-k8s/blob/main/CONTRIBUTING.md#adding-a-changelog-entry)

### CODE REVIEW

File: charts/consul/values.yaml [hunk 1]
```diff
@@ -72,7 +72,7 @@ global:
   # image: "hashicorp/consul-enterprise:1.10.0-ent"
   # ```
   # @default: hashicorp/consul:<latest version>
-  image: docker.mirror.hashicorp.services/hashicorppreview/consul:1.17-dev
+  image: hashicorp/consul-enterprise:1.16
```
Comment (nathancoleman) [inline charts/consul/values.yaml:75]:
```suggestion
  image: hashicorp/consul-enterprise:1.16-ent
```

Comment (aahel) [reply charts/consul/values.yaml:75]: image: hashicorp/consul-enterprise:1.16-ent 
this dosent work.. it internally appends -ent. so it becomes hashicorp/consul-enterprise:1.16-ent-ent. which is invalid.

Comment (nathancoleman) [reply charts/consul/values.yaml:75]: That's interesting. The acceptance test failures are trying to pull without the `-ent` and failing (logs [here](https://github.com/hashicorp/consul-k8s-workflows/actions/runs/5693468452/job/15432801654))

Comment (aahel) [reply charts/consul/values.yaml:75]: maybe its due to the control plane image being used. mine is built from this branch.

**Pull request closed without merging**